### PR TITLE
$products_id is set twice

### PR DESCRIPTION
**Pull request by Zen4All**
$products_id is set on line 34 and 58

[Review on Reviewable](https://reviewable.io/reviews/zencart/zc-v1-series/316)

### CODE REVIEW

File: admin/specials.php [hunk 1]
```diff
@@ -55,7 +55,6 @@
         $specials_date_available = ((zen_db_prepare_input($_POST['start']) == '') ? '0001-01-01' : zen_format_date_raw($_POST['start']));
         $expires_date = ((zen_db_prepare_input($_POST['end']) == '') ? '0001-01-01' : zen_format_date_raw($_POST['end']));
 
-        $products_id = zen_db_prepare_input($_POST['products_id']);
```
Comment (drbyte) [inline admin/specials.php]: Agreed. This was a duplicate of line 34.

**Comment by drbyte** [comment]
:+1:

**Comment by ajeh** [comment]
:+1:

**Comment by ajeh** [comment]
As a heads up, when changing files it is also a good idea to fix the header copyright to the current year.

And if the headers are the old style, they should also be brought up to date.

**Comment by Zen4All** [comment]
:+1:

**Pull request merged**